iOSClient/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -206,7 +206,7 @@
 
 "_optimized_photos_"            = "Optimized photo resolution";
 "_upload_del_photos_"           = "Remove files after upload";
-"_optimized_photos_how_"        = "If your iPhone is low on space, full-resolution photos are automatically replaced with optimized versions. The full-resolution versions are stored in your cloud.";
+"_optimized_photos_how_"        = "If your iPhone is low on space, full-resolution photos are automatically replaced with optimized versions. The full-resolution versions are stored in your Nextcloud.";
```

Comment:
I think this should be "If your **device** is low on space", since this app is also meant to be run on iPad (and other - currently not existing iOS devices) 😉